packages/Avatar/src/Avatar.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Avatar.styles";
+
+const propTypes = {
+  /** Avatar content. It can be initial as a string or icon */
+  children: PropTypes.node,
+  /** Background color of the Avatar */
+  backgroundColor: PropTypes.string.isRequired,
+  /** Color for the initial or icon */
+  color: PropTypes.string.isRequired,
+  /** Size of Avatar */
+  size: PropTypes.oneOf(ShirtSizes.LIMITED),
+};
+
+const defaultProps = {
+  children: null,
+  size: "medium",
+};
+
+function Avatar(props) {
+  const { backgroundColor, children, size, color } = props;
+
+  const avatarProps = {
+    backgroundColor,
+    children,
+    size,
+    color,
+  };
+
+  return (
+    <sc.Avatar
+      style={{ backgroundColor: avatarProps.backgroundColor, color: avatarProps.color }}
```

Comment:
why are we using style here? shouldn't this be props pass down to `<sc.Avatar />`

Reply:
didn't pass the props in the styles.js. Will update this one.

Reply:
you can add a `$` like $backgroundColor and $color if those props are pass-down the HTML element. 

backgroundColor and color used to be valid attributes in previous HTML versions.

Reply:
ex
https://codesandbox.io/s/avatar-inrdr?file=/src/App.js